src/Fantomas.Core/FormatConfig.fs

```diff
@@ -25,6 +25,24 @@ type MultilineFormatterType =
         | "number_of_items" -> Some(box MultilineFormatterType.NumberOfItems)
         | _ -> None
 
+type MultilineBracketStyle =
+    | Classic
+    | Aligned
+    | ExperimentalStroustrup
+
+    static member ToConfigString(cfg: MultilineBracketStyle) =
+        match cfg with
+        | Classic -> "classic"
+        | Aligned -> "aligned"
+        | ExperimentalStroustrup -> "experimental_stroustrup"
+
+    static member OfConfigString(cfgString: string) =
+        match cfgString with
+        | "classic" -> Some(box Classic)
```

Comment:
I don't think you need to box here, `EndOfLineStyle` doesn't do it, `MultilineFormatterType` does.
Let's settle on not boxing here and always box in the `EditorConfig.fs` code.